lib/Hakyll/Core/Runtime.hs

```diff
@@ -6,12 +6,12 @@ module Hakyll.Core.Runtime
 
 --------------------------------------------------------------------------------
 import           Control.Concurrent.Async.Lifted (forConcurrently_)
-import           Control.Concurrent.STM          (atomically, modifyTVar', readTVarIO, newTVarIO, TVar)
+import           Control.Concurrent.MVar         (modifyMVar_, readMVar, newMVar, MVar)
```

Comment:
Does that mean we can stop depending on `array`, `lifted-async` and `stm` that were added in #844?

Reply:
We still use `array` for `assocs` and the `forConcurrently_` from `async` is specialised to `IO` instead of having the more general type we need here, so I think we still want `lifted-async`. I removed `stm` from `hakyll.cabal` but it's a boot package (as is `array`) and also a dependency of `lifted-async` so this doesn't actually reduce our dependency burden at all.

Reply:
Ah, pity. Well, at least we don't have to maintain the version bounds on it :) Thanks!